readme.md

```diff
@@ -39,3 +39,14 @@ This will create:
 - .gitignore
 
 in your current working directory.
+
+
+## Build & Scope an element
+
+To uglify and scope a custom element so that it's usuable for the browser, run the following:
+
+```CLI
+create-element --buid [element-name]
```

Comment:
Can we not have the `create-element` command find the element file? It seems a little counter intuitive to have it build every time. Plus, we might wanna set something more customizable up, like a basic gulp build or something? Maybe we make it create files called like `my-cool-element.element.js` or something, so they're easier to find?